templates/security/index.html

```diff
@@ -351,77 +349,6 @@ <h2>Find out why the UK Government puts Ubuntu in first place for security</h2>
-              alt="",
-              height="28",
-              width="32",
-              hi_def=True,
-              loading="lazy",
-              attrs={"class": "p-heading-icon__img p-heading-icon__img--small"}
-            ) | safe
-          }}
-          <h4 class='p-heading-icon__title'>White paper</h4>
-        </div>
-      </div>
-      <h3 class='p-heading--4'><a href='/engage/a-small-to-medium-size-business-guide-to-cybersecurity' onclick='dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External link', 'eventAction' : 'Security - left', 'eventLabel' : 'White paper - Read Ubuntu Core Security Whitepaper', 'eventValue' : '1' });'>A small to medium-size business guide to cybersecurity</a></h3>
-      <!-- rtp section end -->
-    </div>
-    <div class="col-4 p-divider__block" id="ubuntu-com_security_center">
-      <!-- rtp section start -->
-      <div class='p-heading-icon--muted'>
-        <div class='p-heading-icon__header'>
-          {{
-            image(
-              url="https://assets.ubuntu.com/v1/b061c401-White+paper.svg",
-              alt="",
-              height="28",
-              width="32",
-              hi_def=True,
-              loading="lazy",
-              attrs={"class": "p-heading-icon__img p-heading-icon__img--small"}
-            ) | safe
-          }}
-          <h4 class='p-heading-icon__title'>White paper</h4>
-        </div>
-      </div>
-      <h3 class='p-heading--4'><a href='https://pages.ubuntu.com/IoT-Security-whitepaper.html' onclick='dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External link', 'eventAction' : 'Security - center', 'eventLabel' : 'White paper - Read IoT Security Whitepaper', 'eventValue' : '1' });'>IoT Security Whitepaper</a></h3>
-      <!-- rtp section end -->
-    </div>
-    <div class="col-4 p-divider__block" id="ubuntu-com_security_right">
-      <!-- rtp section start -->
-      <div class='p-heading-icon--muted'>
-        <div class='p-heading-icon__header'>
-          {{
-            image(
-              url="https://assets.ubuntu.com/v1/b061c401-White+paper.svg",
-              alt="",
-              height="28",
-              width="32",
-              hi_def=True,
-              loading="lazy",
-              attrs={"class": "p-heading-icon__img p-heading-icon__img--small"}
-            ) | safe
-          }}
-          <h4 class='p-heading-icon__title'>Whitepaper</h4>
-        </div>
-      </div>
-      <h3 class='p-heading--4'><a href='https://assets.ubuntu.com/v1/66fcd858-canonical-ubuntu-core-security-2018-11-13.pdf' onclick='dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'product page', 'eventAction' : 'gateways - right', 'eventLabel' : 'A small to medium-size business guide to cybersecurity', 'eventValue' : '1' });'>Ubuntu Core Security Whitepaper</a></h3>
-      <!-- rtp section end -->
-    </div>
-  </div>
-</section>
-
 <section class="p-strip is-deep">
```

Comment:
A strip has been removed here, so there are now two consecutive strips with white backgrounds, where they should alternate down the rest of the page between `p-strip--light` and `p-strip`